explorer/app/components/Project/components/Section/section.styles.ts

```diff
@@ -1,8 +1,18 @@
+import { SectionTitle as Title } from "./components/SectionTitle/sectionTitle";
 import styled from "@emotion/styled";
-import { ButtonBase, Typography } from "@mui/material";
 import { BREAKPOINT } from "../../../../hooks/useBreakpointHelper";
+import { ButtonBase, Typography } from "@mui/material";
 
 export const Section = styled.div`
```

Comment:
Added basic non collapsable section. Will update names of collapsable section styles with the refactor of section.

Reply:
Apologies for not getting a ticket to you for a TODO here. I have created a stub ticket [here](https://github.com/clevercanary/data-browser/issues/400); can you add a corresponding TODO?

Reply:
Done